### DISCUSSION TITLE
EVBox Elvi setup

### DISCUSSION THREAD

**Discussion by weemaba999** [lbbrhzn/ocpp#246] [General]
Hi all,

I managed to install the OCPP integration in home assistant and downloaded the EVBox connect app.
After connecting to the Charger, entering the seciurity code, I entered the installation modus

I have the possibility to change the APN, Faserotation, charging power, charging management platform, etc....

As documented, I find a way to enter the ws://homeassistant.local:9000 string into the charging management platform.

After entering the string, the SAVE button stays grayed out.... I tried to enter IP-adres instead of dns name, but the button keeps greyed out...

Does anyone recognize this issue ? I'm not able to save the new configuration in my EVBox charging app

Thanks very much in advance !

Kr,

Bart

**Comment by drc38** [comment]
Have you been through this thread #221 which has instructions on the Elvi setup?

**Comment by dreimer1986** [comment]
> 

You forgot the / at the end. " ws://homeassistant.local:9000/"

**Reply by weemaba999** [reply]
I succeeded in getting the information from my charger, however, my billing engine, the one that was formerly connected as the management platform doesn't report my charging anymore. Is it possible to connect both ? HA & EVBox ?

Thanks in advance,

Kr,

Bart

**Reply by lbbrhzn** [reply]
Hi @weemaba999,  that will depend on your charger. With the Wallbox Pulsar this is possible: It will always keep the connection to its original management platform; the OCPP connection is optional and additional. Other chargers will behave differently, and only support a single management connection.

The OCPP standard inherently assumes that a charge point only has a connection to a single central server. Maintaining multiple OCPP connections to several central systems is a recipe for disaster: which server will approve a transaction to initiate charging? How should a central server behave if it receives a notification wrt to a transaction it did not approve? What if central severs set different charging limits?

Now, if the original management platform also uses OCPP you might think you could create a proxy, relaying messages between the management platform and charge point and parsing them to display the information in home assistant. In theory this could work, however, it would require you to know the websocket url and the credentials (identity, password) required to connect to the original management platform.  So, there's where theory hits a practical limit: In general, these credentials are meant to be kept secret, and assuming the charge point is properly secured, they will be.

**Reply by dreimer1986** [reply]
Our charger supports ONLY one OCPP server being set. In my case Everon as default OR the custom server from this addon. Not both at the same time.

**Comment by dreimer1986** [comment]
Does your Elvi output Energy.Active.Import.Register value aka the current you suck out of the box right now? If so, could you try if this https://github.com/lbbrhzn/ocpp/issues/221 works for you? Can you set a currency limit with the addon? I can't but I am almost sure it's due to the missing parts for current detection and a artificial firmware block.

**Comment by jay-jay-s** [comment]
I'm also having difficulties integrating EvBox Elvi to Home Assistant. I added the ws://homeassistant.local:9000/ and also tried with my Home Assistant ip, like ws://192.168.1.xxx:9000/. The OCPP integration doesn't get any data from the charger. Any suggestions?

**Reply by dreimer1986** [reply]
Yes, is the wallbox in the SAME IP range as Home Assistant is? No castrated WIFI in any ways, like UBIQUITI only allowing to access the Gateway and this is it? (They call that Guest Network). Depending on your answers you might need a bit more tinkering to make them see each other.

**Reply by Gambino88** [reply]
Hey, did u find out the cause? I`m also having this issue. Thanks.

**Reply by dreimer1986** [reply]
Some pics of the setup in the EVbox app maybe? Something must be wrong from there to your Home Assistant. It works perfect over here. Do you see the EVBox in your network, like on your router?

**Reply by jay-jay-s** [reply]
Sorry, forgot about this. It turned out that my wifi connection was extremely poor. It's now sorted out with a mesh network.

**Reply by dreimer1986** [reply]
Yeah, the Elvi Box has terrible WIFI. I had to get closer by Mesh, too.

**Comment by Gambino88** [comment]
Hey, thanks for the reply

Yeah I see it on the router.

Here are some pics of the app config (attached)

Don't really get why it's showing as offline mode, it's connected to the
wifi, it has the same starting address as the home assistant.

If you want we can move this conversation on messenger or discord, I dunno.

Thanks again for your help.

I'm thi guy on Facebook if you want a simpler communication

**Reply by dreimer1986** [reply]
Mail attachments don't show up here as it seems ^^

**Reply by Gambino88** [reply]
![IMG_0340](https://github.com/lbbrhzn/ocpp/assets/152916691/f848505d-b042-4008-bfb8-6d0956e3531d)
![IMG_0341](https://github.com/lbbrhzn/ocpp/assets/152916691/a7595d94-2304-40d7-b9c3-7c91800d09f1)
![IMG_0342](https://github.com/lbbrhzn/ocpp/assets/152916691/b0304847-173c-46c5-9952-8064cf8b3b67)
didnt know that. here you go :D

**Reply by dreimer1986** [reply]
As first step, use the IP of the Home Assistant instance as test, then Select and then Save. After a reboot of the Box try again. Firmware is recent as it seems.

**Reply by Gambino88** [reply]
i keep getting a connection error everytime i try to save the charging pltform setting in installer mode, dont even know how i managed to do it in the first place. ill get back to you once i figure it out

**Reply by Gambino88** [reply]
I managed to input the IP address, still no go

**Reply by dreimer1986** [reply]
Can you ping the IP of the Box from a PC or something in the same network? If the Setting in there is correct the only flaw you can still have is a connection problem. The Box works fine here for Months now.

**Reply by Gambino88** [reply]
yea, ping works, laggy, but it works
![image](https://github.com/lbbrhzn/ocpp/assets/152916691/bee6615a-0274-43d8-b8c6-e9c6193c6faa)

**Reply by dreimer1986** [reply]
I am quite sure your problem is the WIFI connection. I have no ping above 100 ms here and even then the Box sometimes goes into unavailable mode.
